composer.json

```diff
@@ -7,23 +7,24 @@
     "prefer-stable": true,
     "require": {
         "drupal/cas": "1.6",
-        "drupal/core": "^8.7",
+        "drupal/core": "^8.9",
         "php": ">=7.2"
     },
     "require-dev": {
         "composer/installers": "~1.5",
-        "drupal/core-composer-scaffold": "^8.8",
+        "drupal/cas_mock_server": "^1.0@alpha",
+        "drupal/core-composer-scaffold": "^8.9",
         "drupal/config_devel": "~1.2",
         "drupal/devel": "~1.2",
         "drupal/drupal-extension": "~4.0",
-        "drush/drush": "~9.0@stable",
+        "drush/drush": "~10.0",
```

Comment:
Can you check if egulias/email-validator is needed?

Reply:
Required by drupal/core but seems like it is ok to remove constraint as not actual anymore.